docs/source/llms/langchain/index.rst

```diff
@@ -345,3 +347,60 @@ How can I log my chain from code?
             :alt: Logging a LangChain model from a code script file
             :width: 100%
             :align: center
+
+
+How can I log an agent built with LangGraph to MLflow?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The LangGraph integration with MLflow is designed to utilize the `Models From Code feature <../../model/models-from-code.html>`_
+in MLflow to broaden and simplify the support of agent serialization.
+
+To log a LangGraph agent, you can define your agent code within a script, as shown below, saved to a file ``langgraph.py``:
+
+.. code-block:: python
+
+    from typing import Literal
+
+    from langchain_core.tools import tool
+    from langchain_openai import ChatOpenAI
+    from langgraph.prebuilt import create_react_agent
+
+    import mlflow
+
+
+    @tool
+    def get_weather(city: Literal["seattle", "sf"]):
+        """Use this to get weather information."""
+        if city == "seattle":
+            return "It's probably raining. Again."
+        elif city == "sf":
+            return "It's always sunny in sf"
+
+
+    llm = ChatOpenAI()
+    tools = [get_weather]
+    graph = create_react_agent(llm, tools)
+
+    # specify the Agent as the model interface to be loaded when executing the script
+    mlflow.models.set_model(graph)
+
+When you're ready to log this agent script definition to MLflow, you can refer to 
+this saved script directly when defining the model:
+
+.. code-block:: python
+
+    import mlflow
+
+    input_example = {
+        "messages": [{"role": "user", "content": "what is the weather in seattle today?"}]
+    }
+
+    with mlflow.start_run():
+        model_info = mlflow.langchain.log_model(
+            lc_model="./langgraph.py",  # specify the path to the LangGraph agent script definition
+            artifact_path="langgraph",
+            input_example=input_example,
+        )
+
+When the agent is loaded from MLflow, the script will be executed and the defined agent will be
+made available for use for invocation.
```

Comment:
should we have an example of the invocation call?

Reply:
added!